common/app/experiments/Experiments.scala

```diff
@@ -123,5 +123,5 @@ object DeeplyRead
       description = "When ON, deeply read footer section is displayed",
       owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
       sellByDate = LocalDate.of(2023, 10, 31),
-      participationGroup = Perc50,
+      participationGroup = Perc0A,
```

Comment:
Set this to Perc0A temporarily as [discussed in the WebX channel](https://chat.google.com/room/AAAAG9rU0m0/nIkTMAoGP9U)